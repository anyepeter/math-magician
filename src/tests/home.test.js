import { render } from '@testing-library/react';
import Home from '../pages/home';

describe('display the home page', () => {
  it('check the contents of the home page', () => {
    const { container } = render(
      <Home />,
    );
    expect(container).toMatchSnapshot();
  });
});
