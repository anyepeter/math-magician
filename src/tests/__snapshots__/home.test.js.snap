// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`display the home page check the contents of the home page 1`] = `
<div>
  <main
    class="main"
  >
    <h2
      class="welcome"
    >
      Welcome to our page
    </h2>
    <div
      class="main-box"
    >
      <p
        class="sub-box"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur. Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.
      </p>
      <p
        class="lores"
      >
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.Lorem ipsum dolor sit amet consectetur adipisicing elit. Laborum maxime a ex quaerat nesciunt nostrum labore blanditiis voluptatum, repellendus sed, odio sit eveniet natus harum nisi voluptatem amet. Consequatur, consectetur.
      </p>
    </div>
  </main>
</div>
`;
